config/routes.rb

```diff
@@ -644,6 +644,34 @@
       ) + adv_search_post + exp_post + save_post
     },
 
+    :container_build          => {
+      :get  => %w(
+        download_data
+        edit
+        index
+        new
+        show
+        show_list
+        tagging_edit
+        tag_edit_form_field_changed
+      ),
+      :post => %w(
+        button
+        create
+        dynamic_checkbox_refresh
```

Comment:
I tried using the UI to access all of them but I couldn't really. All I know is that this list is the same as for all the other entities so if something is not relevant then it might not be relevant for other entities.